code/workspaces/web-app/src/components/app/AssetRepoSideBar.js

```diff
@@ -0,0 +1,23 @@
+import { withStyles } from '@material-ui/core/styles';
+import List from '@material-ui/core/List';
+import PropTypes from 'prop-types';
+import React from 'react';
+import SideBarGroup from './SideBarGroup';
+import SideBarButton from './SideBarButton';
+import sideBarStyles from './sideBarStyles';
+
+const AdminSideBar = ({ classes }) => (
```

Comment:
Should this be called `AssetRepoSideBar`?

Reply:
Thanks, done.